src/pages/IncomePage.jsx

```diff
@@ -1,5 +1,38 @@
-function Income() {
-  return <h2>Income</h2>;
-}
+import Table from "../components/Table";
+
+const rows = [
+  { id: 1, name: "IGN1", description: "Description" },
+  { id: 2, name: "IGN2", description: "12345" },
+  { id: 3, name: "IGN3", description: "11112345" },
+];
+const columns = [
```

Comment:
We could extract somewhere table configuration so that our component will be more readable and just import it from another place. You can create maybe a common folder where you can have different reusable things such as table config for example.